src/app/core/services/portal.service.ts

```diff
@@ -106,4 +107,29 @@ export class PortalService {
   getCostCenters(): Observable<ICostCenter[]> {
     return this._portalApi.getCostCenters();
   }
+
+  getLibraryTypeOptions(): Observable<ILibraryTypeOption[]> {
+    return this._portalApi.getLibraryTypeOptions();
+  }
+  getLibraryTypeOption(id: number): Observable<ILibraryTypeOption> {
```

Comment:
Please add spacing please between code. Do the same to others.

Reply:
Kindly see update